.github/workflows/tests.yml

```diff
@@ -159,6 +159,18 @@ jobs:
       - uses: Swatinem/rust-cache@v2
       - run: cargo test --features=std
 
+  windows7:
+    name: Windows 7 Test
+    runs-on: windows-2022
```

Comment:
We should add a note that this tests the windows 7 target build, but the actual runtime environment is still like Windows 10+.

Reply:
Renamed the test to indicate this.